src/components/Footer/Address/Email/Email.js

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import copyStringToClipboard from '../../../../utils/copyStringToClipboard';
+import Toast from './Toast/Toast';
+
+const Email = ({ className, email }) => {
+  const emailRef = useRef();
+  const [isToastVisible, setIsToastVisible] = useState(false);
+
+  const handleClickEmail = event => {
+    if (!matchMedia('(pointer:fine)').matches) {
+      return;
+    }
+
+    copyStringToClipboard(event, email, setIsToastVisible);
+  };
```

Comment:
Is this an "Is this desktop" check? Reads like this url copies and opens the email client on non mobile